Objects/unicodeobject.c

```diff
@@ -15228,13 +15228,16 @@ PyDoc_STRVAR(unicode_doc,
 "str(object='') -> str\n\
 str(bytes_or_buffer[, encoding[, errors]]) -> str\n\
```

Comment:
```suggestion
str(bytes_or_buffer, encoding='utf-8', errors='strict') -> str\n\
```

Reply:
I would read it as the defaults applying in all cases including when a single positional arg is given. Even though docs below clarify that's not the case, the users are used to the idea that just by reading the signature is enough to understand what will be the default arg values.

It seems like in this case where defaults are applied in a weird way, it's best to undersupply information in the signature which will force users to either read the docs below or to supply desired values for these args, - in either case it will work as expected.